pkg/controller/ibmblockcsi/ibmblockcsi_controller.go

```diff
@@ -525,8 +525,11 @@ func (r *ReconcileIBMBlockCSI) reconcileClusterRole(instance *ibmblockcsi.IBMBlo
 			logger.Error(err, "Failed to get ClusterRole", "Name", cr.GetName())
 			return err
 		} else {
-			// Resource already exists - don't requeue
-			//logger.Info("Skip reconcile: ClusterRole already exists", "Name", cr.GetName())
+			err = r.client.Update(context.TODO(), cr)
```

Comment:
Is it possible to cover this with a unit test?

Reply:
sure, but not trivial.
I opened CSI-3059